src/components/Onboarding.tsx

```diff
@@ -222,4 +236,6 @@ export default function HomePage() {
 			</Box>
 		</Box>
 	);
-}
+};
+
+export default Onboarding; // Add this line to export the component;
```

Comment:
I have tested the redirect functionality and it works as expected. However the concern I have is that now  
localhost:3000/ and id8.guru/ seem to be giving a 404 error. Before this PR they were redirecting to what you have now put as the onboarding page. 

My advice is to maintain the / as it was and simply redirect the users who have both the project and profile straight to /home.